includes/ets_user_qa_function.php

```diff
@@ -195,176 +198,210 @@ public function ets_ask_qustion_tab() {
+							echo ($value['date']); ?></h6></td>
 						</tr>
-
 						<?php 
-						if(!empty($value['answer'])){?>
+						if(!empty($value['answer'])){
+						?>
 							<tr>
 								<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
 								<td colspan="2"><p> <?php echo $value['answer'];?></p></td> 
 							</tr> 
-							<?php 
-						} else { ?>
+						<?php 
+						} else {
+						?>
 							<tr>
-								<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
-								<td colspan="2" class="ets-no-answer"><h6><p><i><?php echo __("Answer awaiting",'ets_q_n_a');?>...</i></p></h6></td>	
+								<td class="ets-question-title"><p><?php echo __('Answer:','ets_q_n_a'); ?></p></td>
+								<td colspan="2" class="ets-no-answer" ><h6><p><i><?php echo __("Answer awaiting",'ets_q_n_a');?>...</i></p></h6></td>	
 							</tr> 
 							<?php
 						}
+						$count++;
+						if($count > $productQaLength){
+							break;
+						}  
 						
-					}
-					?> 
-					</table>
-					</div>
+					} ?>
+					</tbody>
+					</table>  
+				</div>
 					<?php
-				} 
+				}
+				 	?>  
+				<button type="submit" id="ets-load-more" class="btn btn-success" name="ets_load_more" value=""><?php echo $loadMoreButtonName; ?></button>
+				<div class="ets_pro_qa_length"><p hidden><?php echo $keyData;?><p></div>
+				<?php
 			}
+		} else {
+			//Show Question Answer Listing Type Table With OUt Load More
+			if(!empty($etsGetQuestion)){ 
+				?>
+				<div class="table-responsive my-table">
+				<table class="table table-striped"> 
+				<?php
+				foreach ($etsGetQuestion as $key => $value) {
+					?> 
+					<tr class="ets-question-top">
+							<td class="ets-question-title"><p><?php echo __('Question','ets_q_n_a'); ?>:</p></td>
+							<td class="ets-question-description"><p><?php echo $value['question'];?></p></td> 
+							<td class="ets-cont-right"><h6 class="user-name"><?php echo $value['user_name'] . "<br>";    
+							echo ($value['date']);
+							?></h6></td>
+					</tr>
+
+					<?php 
+					if(!empty($value['answer'])){?>
+						<tr>
+							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
+							<td colspan="2"><p> <?php echo $value['answer'];?></p></td> 
+						</tr> 
+						<?php 
+					} else { ?>
+						<tr>
+							<td class="ets-question-title"><p><?php echo __('Answer','ets_q_n_a'); ?>:</p></td>
+							<td colspan="2" class="ets-no-answer"><h6><p><i><?php echo __("Answer awaiting",'ets_q_n_a');?>...</i></p></h6></td>	
+						</tr> 
+						<?php
+					}
+					
+				}
+				?> 
+				</table>
+				</div>
+				<?php
+			} 
+		}
 			?> 
 		<div class="ets-question-detail-ajax" id="ets-question-detail-ajax"></div>
-  		 
+			 
 		<?php
+
+	}
+
+	/**
+	* Create shortcode for QA listing.
+	*/
+	public function display_qa_listing_shortcode($atts){
+		global $product; 
+		if($product){
```

Comment:
We wrapped the whole code in this condition, this will not on other pages than product pages, we have added 'product_id' parameter in the shortcode so that we can call this shortcode anywhere to show the QA of that product.